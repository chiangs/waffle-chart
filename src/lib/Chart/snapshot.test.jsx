import React from 'react';
import renderer from 'react-test-renderer';
import { describe, expect, test } from 'vitest';
import { Chart } from './index';

describe('Chart', () => {
    test('Chart component renders correctly', () => {
        const props = {
            itemProps: [],
            verticalFill: 'top',
            horizontalFill: 'left',
            isAnimatedFill: true,
        };
        const component = renderer.create(<Chart {...props} />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
