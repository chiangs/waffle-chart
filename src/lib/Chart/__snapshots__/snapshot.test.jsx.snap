// Vitest Snapshot v1

exports[`Chart > Chart component renders correctly 1`] = `
<div
  className="waffle-chart top left"
  data-testid="chart-container"
/>
`;
